src/tunacode/tools/glob.py

```
@@ -147,6 +147,120 @@ def _expand_brace_pattern(pattern: str) -> list[str]:
+    if comp.match(rel_path):
+        return True
+    if orig.startswith("**/"):
+        return fnmatch.fnmatch(entry_name, orig[3:])
+    return False
+
+
+def _entry_matches_any_pattern(
+    entry_name: str,
+    rel_path: str,
+    compiled: list[tuple[str, re.Pattern[str]]],
+    recursive: bool,
```

Ruff (0.14.14)

[error] Boolean-typed positional argument in function definition

(FBT001)